lms/djangoapps/courseware/tests/test_discussion_xblock.py

```diff
@@ -225,30 +225,32 @@ def test_studio_view(self):
         self.assertIn('data-discussion-id="{}"'.format(self.discussion_id), fragment.content)
 
     @ddt.data(
+        (True, True, True),
+        (True, True, False),
         (True, False, False),
+        (False, True, True),
         (False, True, False),
         (False, False, True),
+        (False, False, False),
```

Comment:
Would still like to see if there's a way to simplify this. If we are just generating all possible combinations of True and False in a tuple of length 3 then maybe we should generate that with code/